README.md

```diff
@@ -1,3 +1,72 @@
-# Scylla Go driver
+# ScyllaDB Go Driver
 
-Experimental high performance Scylla driver in Go.
+This is a high-performance client-side driver for [ScyllaDB] written in pure Go.
+
+**Note: this driver is currently in alpha. Bug reports and pull requests are welcome!**
+
+## Installation
+`go get github.com/scylladb/scylla-go-driver`
+## Examples
+```go
+ctx := context.Background()
+
+cfg := scylla.DefaultSessionConfig("exampleks", "192.168.100.100")
+session, err := scylla.NewSession(ctx, cfg)
+if err != nil {
+	return err
+}
+defer session.Close()
+
+requestCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
+defer cancel()
+
+q, err := session.Prepare(requestCtx, "SELECT id, name FROM exampleks.names WHERE id=?")
+if err != nil {
+	return err
+}
+
+res, err := q.BindInt64(0, 64).Exec(requestCtx)
+```
+
+Please see the full [example](examples/basic.go) program for more information.
+
+All examples are available in the [examples](examples) directory
+
+## Features and Roadmap
+
+The driver supports the following:
+
+* Token-aware routing
+* Shard-aware routing (specific to ScyllaDB)
+* Prepared statements
+* Query paging
+* Compression (LZ4 and Snappy algorithms)
+* CQL binary protocol version 4
+* Configurable load balancing policies
+* TLS support
+* Configurable retry policies
+* Authentication support
+* CQL tracing
+* Session and query context support
+
+Ongoing efforts:
+* Batch statements
+* CQL Events
+* Gocql drop-in replacement
+* More tests
+* More benchmarks
+
```

Comment:
Maybe add a section about limitations, missing sections, what could be implemented in the future.